server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/diskusage/IndexDiskUsageAnalyzerIT.java

```diff
@@ -100,7 +100,6 @@ public void testSimple() throws Exception {
             .setMapping(mapping)
             .setSettings(Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, between(1, 5)))
             .get();
-        ensureGreen(index);
```

Comment:
Should we wait for yellow before indexing below?

Reply:
By default we [wait](https://github.com/elastic/elasticsearch/blob/12ad399c488f0cc60e19b5e1b29c6d569cb4351a/client/rest-high-level/src/main/java/org/elasticsearch/client/indices/CreateIndexRequest.java#L309-L322) for all primary copies active when creating an index.